hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -240,11 +250,28 @@ private boolean displayTable(ManagedRocksIterator iterator,
       return displayTable(iterator, dbColumnFamilyDef, out(), schemaV3);
     }
 
+    // If there are no parent directories, create them
+    File dirFile = new File(fileName);
+    if (!dirFile.exists()) {
+      boolean flg = dirFile.mkdirs();
```

Comment:
If we not give the `--max-records-per-file` we need not create the directory, in this scenario, `filename` should be a file.